docs/content/SideNav.md

```diff
@@ -0,0 +1,157 @@
+    <Text>With a label</Text>
+    <Label bg='blue.5'>label</Label>
+  </SideNav.Link>
+  <SideNav.Link href="#url" variant="full">
+    <Text>With a counter</Text>
+    <CounterLabel>16</CounterLabel>
+  </SideNav.Link>
+  <SideNav.Link href="#url">
+    <Heading as="h5" fontSize={1}>A heading</Heading>
+    <Text>and some more content</Text>
+  </SideNav.Link>
+</SideNav>
+```
+
+## Lightweight variant
+
+Add the `variant="lightweight"` prop to `SideNav` to render an alternative, more lightweight version that has items with no borders and are more condensed.
+
+```jsx live
+<BorderBox p={3} backgroundColor='gray.0' maxWidth={360}>
+  <BorderBox border='none' borderBottom={1} mb={2} pb={1}>
+    <Heading as="h5" fontSize={1} color="gray.7">Menu</Heading>
+  </BorderBox>
+  <SideNav variant="lightweight">
+    <SideNav.Link href="#url">
+      <Text>Account</Text>
+    </SideNav.Link>
+    <SideNav.Link href="#url" selected>
+      <Text>Profile</Text>
+    </SideNav.Link>
+    <SideNav.Link href="#url">
+      <Text>Emails</Text>
+    </SideNav.Link>
+    <SideNav.Link href="#url">
+      <Text>Notifications</Text>
+    </SideNav.Link>
+  </SideNav>
+</BorderBox>
+```
+
+It can also appear nested, as a sub navigation. Use margin/padding [System Props](/system-props) to add indentation.
+
+```jsx live
+<SideNav bordered maxWidth={360}>
+  <SideNav.Link href="#url">
+    <StyledOcticon mr={2} size={16} icon={Icons.Person} />
+    <Text>Account</Text>
+  </SideNav.Link>
+  <SideNav.Link href="#url" selected>
+    <StyledOcticon mr={2} size={16} icon={Icons.Octoface} />
+    <Text>Profile</Text>
+  </SideNav.Link>
+
+  <SideNav bordered variant="lightweight" py={3} pl={6} backgroundColor="white">
+    <SideNav.Link href="#url" selected>
+      <Text>Sub item 1</Text>
+    </SideNav.Link>
+    <SideNav.Link href="#url">
+      <Text>Sub item 2</Text>
+    </SideNav.Link>
+    <SideNav.Link href="#url">
+      <Text>Sub item 3</Text>
+    </SideNav.Link>
+  </SideNav>
+
+  <SideNav.Link href="#url">
+    <StyledOcticon mr={2} size={16} icon={Icons.Mail} />
+    <Text>Emails</Text>
+  </SideNav.Link>
+</SideNav>
+```
+
+## Usage with React Router
+
+If using React Router, you can use the `as` prop to render the element as a `NavLink`. React Router will automatically handle setting `aria-current="page"` for you.
+
+```
+<SideNav.Link as={NavLink} to="...">...</SideNav.Link>
+```
+
+## System props
+
+`SideNav` components get `COMMON` and `LAYOUT` system props; using the `bordered` property adds access to `BORDER` props. `SideNav.Link` components get `COMMON` and `TYPOGRAPHY` system props. Read our [System Props](/system-props) doc page for a full list of available props.
+
+## Component props
+
+### SideNav
+
+| Name | Type | Default | Description |
+| :- | :- | :-: | :- |
+| as | String | 'nav' | Sets the HTML tag for the component. |
+| bordered | Boolean | false | Renders the component with a border and gives access to `BORDER` system props. |
```

Comment:
Feels weird to have access to a set of system props behind a boolean 🤔 What do you think about always allowing access to those system props, and just setting a standard border with this prop?